### ISSUE TITLE
examples show bad practive

### ISSUE THREAD

**Issue by ChristianSch** [ChristianSch/skml#12]
they don't show train/test split, but instead just use the full training data as evaluation.

**Comment by ChristianSch** [comment]
turns out stratified train_test_split doesn't work with multi label data (see [here](https://stats.stackexchange.com/questions/65828/how-to-use-scikit-learns-cross-validation-functions-on-multi-label-classifiers)).

**Comment by ChristianSch** [comment]
Depends on #4
